.eslintrc.cjs

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    es2022: true,
+    node: true,
+    jest: true,
+    'jest/globals': true
```

Comment:
Do you mind following the [**.eslintrc.js** in **forms-manager**](https://github.com/DEFRA/forms-manager/blob/main/.eslintrc.cjs)?

E.g. We don't want `commonjs: true` in ES modules or `browser: true` in non-browser files

You'll see how to configure `@typescript-eslint/parser` to pick up **tsconfig.json** too

Reply:
done